nano/node/json_handler.cpp

```diff
@@ -283,6 +283,40 @@ nano::amount nano::json_handler::amount_impl ()
 	return result;
 }
 
+nano::amount nano::json_handler::decimal_amount_impl ()
```

Comment:
Let's document the parsing rules. What are the parsing rules?
Let's make them very strict and very simple.

Reply:
Should ".5" be legal value to parse?

Reply:
Do we really want to parse negative numbers?

Reply:
What is the largest number we support?
What is smallest number?
What happens when a value is too small or too large?

Reply:
Do you want me to document the rules in the code as a comment or somewhere else?

Reply:
The rules will need to be in the documentation to the RPC call. In the mean time, you could add them to the code as a comment.

Reply:
What about whitespace? What are the rules, whitespace allowed?
I would lean on being strict on no whitespace allowed, for simplicity.